Documentation/security/policy/language.rst

```diff
@@ -754,25 +754,45 @@ which is defined as follows:
 .. code-block:: go
 
         // ICMPField is a ICMP field.
+        //
+        // +deepequal-gen=true
+        // +deepequal-gen:private-method=true
         type ICMPField struct {
-        	// Family is a IP address version.
-        	// Currently, we support `IPv4` and `IPv6`.
-        	// `IPv4` is set as default.
-        	//
-        	// +default=IPv4
-        	// +optional
-        	Family string `json:"family,omitempty"`
-
-        	// Type is a ICMP-type.
-        	// It should be 0-255 (8bit).
-        	Type uint8 `json:"type"`
+            // Family is a IP address version.
+            // Currently, we support `IPv4` and `IPv6`.
+            // `IPv4` is set as default.
+            //
+            // +kubebuilder:default=IPv4
+            // +kubebuilder:validation:Optional
+            // +kubebuilder:validation:Enum=IPv4;IPv6
+            Family string `json:"family,omitempty"`
+        
+            // Type is a ICMP-type.
+            // It should be an 8bit code (0-255), or it's name (for example, "Echo Reply").
+            // Allowed ICMP types are:
+            //     Ipv4: Echo Reply | Destination Unreachable | Redirect | Echo | Echo Request |
+            //				   Router Advertisement |Router Selection |Time Exceeded |Parameter Problem |
+            //					 Timestamp | Timestamp Reply | Photuris | Extended Echo Request | Extended Echo Reply
+            //     Ipv6: Destination Unreachable | Packet Too Big | Time Exceeded | Parameter Problem |
+            //					 Echo Request | Echo Reply | Multicast Listener Query| Multicast Listener Report |
+            // 					 Multicast Listener Done | Router Solicitation | Router Advertisement | Neighbor Solicitation |
+            // 					 Neighbor Advertisement | Redirect Message | Router Renumbering | ICMP Node Information Query |
+            // 					 ICMP Node Information Response | Inverse Neighbor Discovery Solicitation | Inverse Neighbor Discovery Advertisement |
+            // 					 Home Agent Address Discovery Request | Home Agent Address Discovery Reply | Mobile Prefix Solicitation |
+            // 					 Mobile Prefix Advertisement | Duplicate Address Request Code Suffix | Duplicate Address Confirmation Code Suffix |
+            // 					 Extended Echo Request | Extended Echo Reply
```

Comment:
The indents here look a bit awkward, can all messages be indented in the same way?